file.go

```diff
@@ -1006,3 +1040,25 @@ func flattenBatchHeaderParse(record string) *BatchHeader {
 	bh.ODFIIdentification = bh.parseStringField(record[79:87])
 	return bh
 }
+
+// flattenIATBatchHeaderParse parses a string of IAT Batch Header data into a IATBatchHeader
+func flattenIATBatchHeaderParse(record string) *IATBatchHeader {
```

Comment:
What's the difference between this and `func (iatBh *IATBatchHeader) Parse(record string) `? It looks like this method duplicates a lot of the logic in Parse.

Reply:
sIATBh is 87 in length.

Reply:
Could we share the index lookups in a function though? I don't want that duplicated.